modules/virt-vm-template-wizard-fields-web.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+// VM wizard includes additional options to VM template wizard
+// Call appropriate attribute in the assembly
+
+[id="virt-vm-wizard-fields-web_{context}"]
+ifdef::virtualmachine[]
+= Virtual machine wizard fields
+endif::[]
+ifdef::vmtemplate[]
+= Virtual machine template wizard fields
+endif::[]
+
+|===
+|Name |Parameter |Description
+
+ifdef::virtualmachine[]
+|Template
+|
+|Template from which to create the virtual machine. Selecting a template will automatically complete other fields.
+endif::[]
+
+
+|Name
+|
+|The name can contain lowercase letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (`.`), or special characters.
+
+|Template provider
+|
+|The name of the user who is creating the template for the cluster or any meaningful name that identifies this template.
+
+|Description
+|
+|Optional description field.
+
+|Operating System
+|
+|The primary operating system that is selected for the virtual machine.
+
+.4+|Boot Source
+|Import via URL (creates PVC)
+|Import content from an image available from an *HTTP* or *S3* endpoint. Example: Obtaining a URL link from the web page with the operating system image.
+
+|Clone existing PVC (creates PVC)
+|Select an existent persistent volume claim available on the cluster and clone it.
+
+|Import via Registry (creates PVC)
+|Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`.
+
+|PXE (network boot - adds network interface)
+|Boot an operating system from a server on the network. Requires a PXE bootable network attachment definition.
+
+|Persistent Volume Claim project
+|
+|Select the project name that you want to use for cloning the PVC.
+
+|Persistent Volume Claim name
+|
+|Select the PVC name that should apply to this virtual machine template if you cloning an existing PVC.
+
+|Flavor
+|Tiny, Small, Medium, Large, Custom
+|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for *Flavor* are determined by the operating system.
+
+.2+|Workload Type
```

Comment:
Why remove "High-performance" from the workload type list?

Reply:
High-performance feature was added back.